src/main/java/org/broadinstitute/hellbender/engine/Variant.java

```diff
@@ -0,0 +1,12 @@
+package org.broadinstitute.hellbender.engine;
+
+/**
+ * Variant is (currently) a minimal variant interface needed by the Hellbender pipeline.
+ * This will be expanded as more methods are needed.
+ */
+public interface Variant {
+    int getStart();
+    int getEnd();
+    boolean isSnp();
+    boolean isIndel();
```

Comment:
Add comments to the interface methods making the contracts clear (eg., mention that `getStart()` and `getEnd()` return 1-based inclusive positions).